src/screens/FearConditioningTrialScreen.tsx

```diff
@@ -111,7 +124,14 @@ export const FearConditioningTrialScreen: ModuleScreen<FearConditioningTrialScre
         />
       </Box>
       {showScale && (
-        <TrialRatingScale onChange={(rating) => setRating(rating)} />
+        <TrialRatingScale
+          onChange={(rating) => {
+            // Update state
+            setRating(rating)
+            // Record time to rate
+            reactionTimeRef.current = Date.now() - startTimeRef.current
```

Comment:
It would be more valuable to record the timestamps. The durations can be a calculated value done on server side.

Reply:
yep, switched it out.